.gitignore

```diff
@@ -1,4 +1,5 @@
 /target
+/bin
```

Comment:
Any idea which tool is creating files in the bin directory?

All generated files from sources go to target (if they get cleaned) or local (if they should not get cleaned with "mvn clean"). Normally, Eclipse and IntelliJ pick this up correctly after being configured from the pom.xml's.

Reply:
Eclipse is; I think I imported the project before installing m2e, then reimport as a Maven project afterwards.

Reply:
ok, let's remove the bin directory from the gitignores.
Anyone accidentally creating it, will then see those files appear, switch to m2e and delete that bin dir, instead of leaving it around and potentially confuse the developers with the target dir.